### PR TITLE
fixes #796

### LINKED ISSUE #796: Getting Started Guide updates
## Opening paragraph
* "Fabric8 Launcher service" -> "Fabric8 Luancher tool"

## Intro
* A prescriptive approach to cloud-native development. + "on OpenShift"
* rm "The four key principles...adhering to cloud-native development."

### OpenShift
* "OpenShift Online is a public cloud hosted and supported by Red Hat" -> "OpenShift Online is a public cloud hosted by Red Hat"
* rm "the" -> "the Red Hat Container Development Kit"

### developers.redhat.com/launch
* rm functional: "It provides a hassle-free way of creating functional example applications..."
* "mission" -> "booster": " It provides a hassle-free way of creating functional example applications, called missions, as well as an easy way to build and deploy those missions to OpenShift."

## SSO federated ID
* update with new steps

## OSO
* update step 3 to say you _can_ deploy zip artifact to OSO
* change step 6 to warning
* step 6 "because the resources allocated to your account are limited." -> "because the resources allocated to your {OpenShiftOnline} account are limited."
* step 6: remove "}"

### PR DESCRIPTION
**Pull request by rhoads-zach**

### CODE REVIEW

File: docs/minishift-installation/master.adoc [hunk 1]
```diff
@@ -18,9 +18,9 @@ include::topics/proc_starting-and-configuring-the-openshiftlocal-for-the-launche
 include::topics/proc_creating-a-github-personal-access-token.adoc[leveloffset=+1]
 
 [[create-launcher-app]]
-== Creating the {launcher} Service
+== Creating the {launcher} Tool
 
-This chapter describes multiple ways of creating the {launcher} service.
+This chapter describes multiple ways of creating the {launcher} tool.
```
Comment (Ladicek) [inline docs/minishift-installation/master.adoc]: This isn't really "creating", is it? I guess we're intentionally careful about wording here, but I think this doesn't describe how to "create" the tool, but how to _deploy_ the dool.

File: docs/topics/appendix-glossary.adoc [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 {launcher-oso}:: link:{link-launcher-oso}[{launcher-oso}] is a standalone getting started experience offered by Red Hat for jumpstarting cloud-native application development on OpenShift. It provides a hassle-free way of creating functional example applications, called missions, as well as an easy way to build and deploy those missions to OpenShift. 
 
-{launcher} Service:: The {launcher} service is the upstream project from which link:{link-launcher-oso}[{launcher-oso}] is based.
+{launcher} Tool:: The {launcher} tool is the upstream project from which link:{link-launcher-oso}[{launcher-oso}] is based.
```
Comment (Ladicek) [inline docs/topics/appendix-glossary.adoc]: Neither the "service" nor the "tool" are the upstream project. IMHO, it's just "Fabric8 Launcher is the upstream project".

File: docs/topics/con_introduction-to-productname.adoc [hunk 3]
```diff
@@ -6,45 +6,23 @@ This section provides a brief overview of the foundational concepts behind {Prod
...
+link:{link-launcher-oso}[{launcher-oso}] is a standalone getting started experience offered by Red Hat for jumpstarting cloud-native application development on OpenShift. It provides a hassle-free way of creating example applications, called boosters, as well as an easy way to build and deploy those boosters to OpenShift. 
 
-.The {launcher} Service
-The {launcher} service is the upstream project from which link:{link-launcher-oso}[{launcher-oso}] is based. You can also install and execute the {launcher} service on your {OpenShiftLocal}. For more information, see the link:{link-launcher-openshift-local-install-guide}#create-launcher-app[Creating the {launcher} Service] chapter of the {minishift-installation-guide-name} guide.
+.The {launcher} Tool
+The {launcher} tool is the upstream project from which link:{link-launcher-oso}[{launcher-oso}] is based. You can also install and execute the {launcher} tool on your {OpenShiftLocal}. For more information, see the link:{link-launcher-openshift-local-install-guide}#create-launcher-app[Creating the {launcher} Tool] chapter of the {minishift-installation-guide-name} guide.
```
Comment (Ladicek) [inline docs/topics/con_introduction-to-productname.adoc]: Again the word "tool" in the first sentence is superfluous IMHO.

File: docs/minishift-installation/master.adoc (hunk 1 above)
Comment (Ladicek) [reply docs/minishift-installation/master.adoc]: Or _install_. I've seen in this PR that we use both.

**Comment by rhoads-zach** [comment]
thanks @Ladicek fixed.

**Comment by rhoads-zach** [comment]
merged: 9f83a1bd8296fa6e7857acf787e720b5ae4ca323

**Pull request closed without merging**